src/Storage/Storage.Autorest/docs/Start-AzStorageAccountMigration.md

```diff
@@ -0,0 +1,312 @@
+---
+external help file:
+Module Name: Az.Storage
+online version: https://learn.microsoft.com/powershell/module/az.storage/start-azstorageaccountmigration
+schema: 2.0.0
+---
+
+# Start-AzStorageAccountMigration
+
+## SYNOPSIS
+Account Migration request can be triggered for a storage account to change its redundancy level.
+The migration updates the non-zonal redundant storage account to a zonal redundant account or vice-versa in order to have better reliability and availability.
+Zone-redundant storage (ZRS) replicates your storage account synchronously across three Azure availability zones in the primary region.
+
+## SYNTAX
+
+### CustomerExpanded (Default)
+```
+Start-AzStorageAccountMigration -AccountName <String> -ResourceGroupName <String> -TargetSku <String>
+ [-SubscriptionId <String>] [-Name <String>] [-Type <String>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait]
+ [-PassThru] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### CustomerViaIdentityExpanded
+```
+Start-AzStorageAccountMigration -InputObject <IStorageIdentity> -TargetSku <String> [-Name <String>]
+ [-Type <String>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-PassThru] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+### CustomerViaJsonFilePath
```

Comment:
Add test cases and examples for JsonPath and JsonString.

Reply:
Done. Added samples for both parameter sets.